netbout-web/src/main/js/bout.js

```diff
@@ -251,6 +251,19 @@ $(document).ready(
           return false;
         }
     );
+  if ($( "#file-binary" )[0].value === "") {
+      $( "#file-submit" ).prop('disabled', true);
+  } else {
+      $( "#file-submit" ).prop('disabled', false);
+  }
+  $( "#file-binary" ).change(function() {
```

Comment:
@kujtimiihoxha let's apply paired brackets rule here too, `function() {` should be on new line